fp.js

```diff
@@ -18,17 +18,25 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(person => {
+people.filter(person => {
   if(person.age < 35){
     peopleYoungerThan35.push(person)
   }
 })
 
-// Convert the following to use `.forEach`
+// // Convert the following to use `.forEach`
 
 var people = [
   {name: "Jane", age: 32},
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-var peopleWhoseNamesHaveTheLetterS = people.filter(person => person.name.match("s"))
+// function findSPeople(person){
+//   var yup = person.name.match("s")
+//   console.log(yup);
+// }
+// people.forEach(findSPeople)
+people.forEach(person => {
+  var yup = person.name.match("s")
+  console.log(yup);
+})
```

Comment:
`forEach` does **NOT** take a return value because it always return `undefined`. Here you'll want to instantiate a new array and make sure to push values into it from within `forEach`.

```js
var people = { ... } // original data goes here
var beginsWithS = []
people.forEach(person => {
  if(person.name.match("s")){
    beginsWithS.push(person);
  }
});
```